examples/index.html

```diff
@@ -253,7 +253,9 @@ <h1><a href="https://threejs.org">three.js</a></h1>
 
 		function escapeRegExp( string ) {
 
-			return string.replace( /[.*+?^${}()|[\]\\]/g, '\\$&' ); // https://stackoverflow.com/a/6969486/5250847
+			string = string.replace( /[.*+?^${}()|[\]\\]/g, '\\$&' ); // https://stackoverflow.com/a/6969486/5250847
+
+			return '(?=.*' + string.split( ' ' ).join( ')(?=.*' ) + ')'; // match all words, in any order
```

Comment:
Do you mind updating the function in `docs/index.html` as well to keep both in sync?